src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+
 body {
   margin: 0;
+
+  color: $main-accent-color;
+
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.product-card {
```

Comment:
I believe the height of your card is wrong and it is causing most fails in the test

Reply:
Thank you, i will make the adjustments.